controller/search_blackbox_test.go

```diff
@@ -705,6 +705,18 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 		assert.Len(t, result.Data, 3) // alice worked on 3 issues in sprint1
 	})
 
+	s.T().Run("space=spaceID AND creator=bob", func(t *testing.T) {
+		filter := fmt.Sprintf(`
+				{"$AND": [
+					{"space":"%s"},
+					{"creator":"%s"}
+				]}`,
+			spaceIDStr, fxt.IdentityByUsername("spaceowner").ID.String())
+		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+		require.NotEmpty(t, result.Data)
+		assert.Len(t, result.Data, 9) // we have 9 items created by bob
+	})
```

Comment:
is `bob` a `space-owner`?

Reply:
Corrected 5fd24915ed831c1a9328296835800d91c4b05a6e